tests/plugins/autoembed/manual/autoembed.md

```diff
@@ -1,3 +1,4 @@
+@bender-tags: feature, generic
```

Comment:
Please remove broken links:
`https://www.google.dk/maps/place/Zygmunta+S%C5%82omi%C5%84skiego+15,+Warszawa` and
`http://tellto.tumblr.com/post/112160646328`
from the list.

Reply:
`http://tellto.tumblr.com/post/112160646328` is no brainer here as the URL no longer exists. However, in the case of Google maps, I'm not sure. It looks like we are just not authorized to use this API but it correctly pings to the provider. I suppose that running tests from authorized domains would work. That means we are actually getting expected behavior.

Also, removing invalid tests should close: https://github.com/ckeditor/ckeditor4/issues/2849 I will add it to the closing footnote. Funny that Spotify started working.